src/ui/portal/src/components/Gramercy.vue

```diff
@@ -4,9 +4,9 @@
       <blockquote class="blockquote mb-0">
         <p>{{message}}</p>
         <footer class="blockquote-footer" v-if="!isAnonymous">
-          <a v-bind:href="twitterUrl" target="_blank">{{senderHandle}}</a>
```

Comment:
Here's an example of how you can simplify this:

``` html
<footer class="blockquote-footer" v-if="!isAnonymous">
    <a v-bind:href="twitterUrl" target="_blank">{{senderHandle}}</a>
    <a v-bind:href="senderTwitterUrl" target="_blank">{{senderHandle}}</a>
</footer>
<footer class="blockquote-footer" v-else>Anonymous</footer> 

 for <a v-bind:href="recipientTwitterUrl" target="_blank">{{recipientHandle}}</a>
```
Notice the `v-if="!isAnonymous"` and `v-else` attributes on the two footer tags.  Only one of those tags will display based on the condition.  However, because the ` for <a ....` snippets was outside of those tags they would both display.  Since we want it to always display we can move it to the end.

Reply:
Oh right, I'd assumed that by not having the "for ..." inside the footer tags that it'd end up with some weird formatting (or not displaying).

Makes sense, will update!